src/Tribe/Metabox.php

```diff
@@ -169,11 +173,14 @@ public function get_panels( $post, $ticket_id = null ) {
 		// Let's create tickets list markup to return
 		$tickets = Tribe__Tickets__Tickets::get_event_tickets( $post->ID );
 
-		$panels = array(
-			'list' => tribe( 'tickets.admin.views' )->template( 'editor/panel/list', array( 'post_id' => $post->ID, 'tickets' => $tickets ), false ),
-			'settings' => tribe( 'tickets.admin.views' )->template( 'editor/panel/settings', array( 'post_id' => $post->ID ), false ),
-			'ticket' => tribe( 'tickets.admin.views' )->template( 'editor/panel/ticket', array( 'post_id' => $post->ID, 'ticket_id' => $ticket_id ), false ),
-		);
+		/** @var Tribe__Tickets__Admin__Views $admin_views */
+		$admin_views = tribe( 'tickets.admin.views' );
+
+		$panels = [
+			'list'     => $admin_views->template( 'editor/panel/list', [ 'post_id' => $post->ID, 'tickets' => $tickets ], false ),
```

Comment:
:no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: f2de0a4e4978475cc6005fc011c5c682</sub>